app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
Is `filters` present on the annotations that get passed in? I'm not sure what that looks like, but I guess you could figure it out from the code in `Index`, maybe?